### PR TITLE
fixed some errors

### PR DESCRIPTION
**Pull request by achterbahn**
I just went by the error messages and changed some things, without knowing exactly what it does. 
Anyway: now it runs, but I still have some trouble. 
But I think this is based on valet using php@7.4 or php@8.1/8.2 respectivly.

### CODE REVIEW

**Comment by JoshuaLuckers** [comment]
Thanks @achterbahn ! I’ll take a look at it later.

Feel free to share more details about the issue you’re still experiencing.

File: src/MODXRevolutionValetDriver.php [hunk 1]
```diff
@@ -44,7 +44,8 @@ public function serves($sitePath, $siteName, $uri)
             return true;
         }
 
-        return $this->isMODXRequest($sitePath, '/');
+        //return $this->isMODXRequest($sitePath, '/');
```
Comment (JoshuaLuckers) [inline src/MODXRevolutionValetDriver.php]: @achterbahn what's the reason for this deleted line? Did it throw an error?

**Comment by achterbahn** [comment]
reversed to your version and, happily it works now. 
Thanks for your time. Hope this is of use for anyone else, too.

**Comment by JoshuaLuckers** [comment]
Thanks a lot for taking time to help me out!

**Pull request merged**